org.freecadweb.FreeCAD.yaml

```diff
@@ -722,10 +722,10 @@ modules:
           # to obtain the revcount for a given gitRef (commit hash, branch or tag) run
           # echo $(($(curl -s 'https://api.github.com/repos/FreeCAD/FreeCAD/compare/120ca87015...'"$gitRef"'' | grep "ahead_by" | sed -s 's/ //g;s/"ahead_by"://;s/,//')+1))
           - |
-            revcount=29177
+            revcount=30029
```

Comment:
should be changed to 29410 before merge, for a test build it doesn't matter.

Reply:
ah right. forgot. done.